src/stactools/sentinel1/grd/constants.py

```diff
@@ -28,10 +50,34 @@
         ProviderRole.PROCESSOR,
         ProviderRole.LICENSOR,
     ],
-    url="https://earth.esa.int/web/guest/home",
+    url="https://earth.esa.int/eogateway",
 )
 
-SENTINEL_POLARISATIONS = {
+SENTINEL_GRD_PROVIDER = pystac.Provider(
+    name="Sinergise",
+    roles=[
+        ProviderRole.HOST,
+        ProviderRole.PROCESSOR,
+        ProviderRole.LICENSOR,
+    ],
+    url="https://registry.opendata.aws/sentinel-1/",
+)
+
+SENTINEL_GRD_TECHNICAL_GUIDE = Link(
+    title="Sentinel-1 Ground Range Detected (GRD) Technical Guide",
+    rel="about",
+    target="https://sentinels.copernicus.eu/web/sentinel/technical-guides/sentinel-1-sar/products-algorithms/level-1-algorithms/ground-range-detected",  # noqa: E501
+)
+
+SENTINEL_GRD_LICENSE = Link(
+    title="Sentinel License",
+    rel="license",
+    target="https://scihub.copernicus.eu/twiki/do/view/SciHubWebPortal/TermsConditions",
+)
+
+SENTINEL_GRD_KEYWORDS = ["ground", "sentinel", "copernicus", "esa", "sar"]
+
+SENTINEL_POLARIZATIONS = {
```

Comment:
Do we need to change the spelling? If we keep it as `SENTINEL_POLARISATIONS`, I _think_ this is a non-breaking PR.

Reply:
Alex and I discussed this yesterday, and I believe I understand what you mean. I had used the same spelling as planetary, but we can certainly keep the spelling consistent with what is already in other places in the package.

Reply:
Oh shoot, I did _not_ realize that the SAR extension uses the `z` version. That's my fault; I think aligning with the SAR extension is the appropriate move, so we should switch to `z`, including the constant name. Mea culpa.

Reply:
Including under RTC or is that outside the scope of this PR?